src/test/scala/com/udemy/statistics/algorithm/MonteCarloSpec.scala

```diff
@@ -37,31 +37,31 @@ class MonteCarloSpec extends FlatSpec with Matchers {
     sampleC -> NonEmptyList(98, 150, 56, -23, 98, 120, 660, -135, 430, -12)
   )
 
-  "dataset winProbability" should "appropriately distribute wins to better datasets" in {
+  "datasets" should "appropriately distribute wins to better datasets" in {
```

Comment:
i think subjects of `should` for these tests should be the name of the function you're testing